rosbag2_storage_default_plugins/src/rosbag2_storage_default_plugins/sqlite/sqlite_storage.cpp

```diff
@@ -75,9 +88,29 @@ void SqliteStorage::open(
   const rosbag2_storage::StorageOptions & storage_options,
   rosbag2_storage::storage_interfaces::IOFlag io_flag)
 {
+  std::vector<std::string> pragmas;
   if (!storage_options.storage_config_uri.empty()) {
-    fprintf(stderr, "going to open config file: %s\n", storage_options.storage_config_uri.c_str());
-    throw std::runtime_error("storage specific config file is not yet implemented.");
+    try {
```

Comment:
nitpick: this might be a nice separate function to keep the length of this `open` down